### PR TITLE
Peatland Module on by default

### PR DESCRIPTION
**Pull request by flohump**
## :bird: Purpose of this PR :bird:

- Peatland Module on by default. Peatland area and related GHG emissions are calculated for each model run. 
- GHG prices protect intact peatlands and incentivize the restoration of degraded peatlands

## :wrench: Checklist for PR creator :wrench:

- [x] Labeling pull request correctly [from the label list](https://github.com/magpiemodel/magpie/labels).

* Low risk : Simple bugfixes (missing files, updated documentation, typos) or Start/output scripts
* Medium risk : New realization / Changes to existing realization / Other changes which don't modify default.cfg
* High risk : New input files (if cfg$input is changed in default.cfg) / Modification to core model (eg. changes in equations, calculations, introduction of new sets etc.) / Other changes in default.cfg
 
- [x] Providing additional information based on PR label

* Low risk : No new model run needed.
* Medium risk : Default run based on the current version of the fork from which PR is made
* High risk
  * Default run from the current develop branch: PEAT_NPI_old
  * Default run based on the current version of the fork from which PR is made: PEAT_NPI_new

Emissions_CO2_Land_Land_use_Change-21
Emissions_CO2_Land_Land_use_Change-22
Resources_Land_Cover_Cropland-30
Resources_Land_Cover_Cropland-31
Resources_Land_Cover_Pastures_and_Rangelands-13
Resources_Land_Cover_Pastures_and_Rangelands-14
Productivity_Landuse_Intensity_Indicator_Tau-26
Productivity_Landuse_Intensity_Indicator_Tau-27

- [x] :chart_with_downwards_trend: Performance loss/gain from current default behavior :chart_with_upwards_trend:
  * Current develop branch default : 0.6 hours
  * This PR's default :  0.56 hours
Unknown-60

- [x] Added changes to `CHANGELOG.md`
- [x] Compilation check (model starts without compilation errors - use `gams main.gms action=c` in model folder for testing).
- [x] No hard coded numbers and cluster/country/region names.
- [x] The new code doesn't contain declared but unused parameters or variables.
- [x] Where relevant, In-code comments added including documentation comments.
- [x] Made sure that documentation created with [`goxygen`](https://github.com/pik-piam/goxygen) is okay (use `goxygen::goxygen()` for testing).
- [x] Changes to [`magpie4`](https://github.com/pik-piam/magpie4) R library for post processing of model output (ideally backward compatible).
- [x] Self-review of my own code.
- [x] For high risk runs: validation of major model indicators - Land-use, emissions, food prices, Tau.  %Delete this line in case it is not a high risk run%

## :warning: Additional notes or warnings :warning:
NA

## :rotating_light: Checklist for RSE reviewer :rotating_light:

- [x] PR is labeled correctly.
- [x] `CHANGELOG` is updated correctly
- [x] No hard coded numbers and cluster/country/region names.
- [x] No unnecessary increase in module interfaces
- [x] All required updates in interfaces (if any) have been properly adressed in the module contracts
- [x] In-code comments and documentation comments are satisfactory.
- [x] model behavior/performance is satisfactory.
- [x] Requested changes (if any) were applied correctly

## :rotating_light: Checklist for MAgPIE reviewer :rotating_light:

- [x] PR is labeled correctly.
- [x] `CHANGELOG` is updated correctly
- [x] No hard coded numbers and cluster/country/region names.
- [x] Changes to the model are scientifically sound
- [x] In-code comments and documentation comments are satisfactory.
- [x] model behavior/performance is satisfactory.
- [x] Requested changes (if any) were applied correctly

### CODE REVIEW

File: config/default.cfg [hunk 1]
```diff
@@ -1043,7 +1043,7 @@ cfg$gms$s56_reward_neg_emis <- 0			# def = 0
 # * Peatland GHG emission pricing
 # * 0=off
 # * 1=on
-cfg$gms$s56_peatland_policy <- 0			# def = 0
+cfg$gms$s56_peatland_policy <- 1			# def = 0
```
Comment (tscheypidi) [inline config/default.cfg:1046]:
```suggestion
cfg$gms$s56_peatland_policy <- 1			# def = 1
```

File: config/default.cfg [hunk 2]
```diff
@@ -1058,7 +1058,7 @@ cfg$gms$c57_macc_version  <- "PBL_2007"                # def = PBL_2007
 # * (off): 	Peatland area and associated GHG emissions are assumed zero
 # * (on): 	Peatland area is initialized with present-day degraded and intact peatland.
 # * 	   	GHG emissions are calculated using IPCC Tier 1 emission factors (2013 Wetland supplement).
-cfg$gms$peatland  <- "off"                     # def = off
+cfg$gms$peatland  <- "on"                     # def = off
```
Comment (tscheypidi) [inline config/default.cfg:1061]:
```suggestion
cfg$gms$peatland  <- "on"                     # def = on
```

**Review by tscheypidi — CHANGES_REQUESTED** [review]
comments need an update in default.cfg (see my suggested changes)

**Review by tscheypidi — APPROVED** [review]

**Review by k4rst3ns — APPROVED** [review]

**Review by k4rst3ns — APPROVED** [review]
Looks all good to me. EU is a bit strange with this high emissions now, but as our emissions are anyway not very stable and correct I guess this is a minor thing
.

**Comment by k4rst3ns** [comment]
Maybe than we should clarify, if the peatland emission factors for EU are too high* or
the validation data is lacking detailed peatland emissions.

* Could be as especially in EUR some peatland conversion happened very long ago (more than 100 years), the emission factors we use overestimate emissions from very old degraded peatlands (?)

**Comment by mishkos** [comment]
> Maybe than we should clarify, if the peatland emission factors for EU are too high* or
> the validation data is lacking detailed peatland emissions.
> 
> (*) Could be as especially in EUR some peatland conversion happened very long ago (more than 100 years), the emission factors we use overestimate emissions from very old degraded peatlands (?)

Good point with overestimation. Where are our peatland emissions factors from, IPCC? We might need to revise those. The emissions in EU are very high now, and with the coupling with REMIND I think we will have issues to justify those levels.

**Comment by flohump** [comment]
> > Maybe than we should clarify, if the peatland emission factors for EU are too high* or
> > the validation data is lacking detailed peatland emissions.
> > (*) Could be as especially in EUR some peatland conversion happened very long ago (more than 100 years), the emission factors we use overestimate emissions from very old degraded peatlands (?)
> 
> Good point with overestimation. Where are our peatland emissions factors from, IPCC? We might need to revise those. The emissions in EU are very high now, and with the coupling with REMIND I think we will have issues to justify those levels.

GHG emissions from peatlands in the EU are about 220 Mt CO2eq/yr 
https://onlinelibrary.wiley.com/doi/10.1002/adsu.202000146

The estimate in MAgPIE for EUR is very close. This is CO2 only, but that's the most important gas here.
Emissions_CO2_Land_Land_use_Change_Peatland

Global CO2 emissions from peatlands compare well to Joosten 2010
https://www.wetlands.org/publications/the-global-peatland-co2-picture/

Reporting of wetland GHG emissions is fragmentented and incomplete.
https://greifswaldmoor.de/files/dokumente/GMC%20Schriften/18-02_Barthelmes_GMC.pdf

**Comment by k4rst3ns** [comment]
Ok, thanks @flohump.

I guess than either we were right for the wrong reasons before or our validation data does not include this peatland emissions.
Do you know more about in what regard the validation data includes peatland emissions?

**Comment by mishkos** [comment]
I'm a bit confused by this level of ~220 MtCO2 per year in the given reference. I checked in the European Environment Agency and the value presently is ~15MtCO2 per yer. The discrepancy is considerable. 

I was generally surprised that peatlands are such a huge source of emissions in the EU, like 25% of total GHG in AFOLU. Do they indeed contribute that much? 

Here is the snapshot from the [EEA data](https://www.eea.europa.eu/data-and-maps/data/data-viewers/greenhouse-gases-viewer) 
image

**Comment by flohump** [comment]
> I'm a bit confused by this level of ~220 MtCO2 per year in the given reference. I checked in the European Environment Agency and the value presently is ~15MtCO2 per yer. The discrepancy is considerable.
> 
> I was generally surprised that peatlands are such a huge source of emissions in the EU, like 25% of total GHG in AFOLU. Do they indeed contribute that much?
> 
> Here is the snapshot from the [EEA data](https://www.eea.europa.eu/data-and-maps/data/data-viewers/greenhouse-gases-viewer)
> image

Degraded peatlands in Germany alone already emit about 53 Mio ton CO2eq/yr.
https://www.umweltbundesamt.de/daten/klima/treibhausgas-emissionen-in-deutschland/emissionen-der-landnutzung-aenderung#nachhaltige-landnutzung-und-forstwirtschaft-

I think the category in the EEA DB in your screenshot includes only emissions from intact wetlands.

**Pull request merged**